src/data-workspace/data-value-mutations/mutation-functions.js

```diff
@@ -28,6 +28,16 @@ const UPLOAD_FILE_MUTATION = {
     data: (data) => data,
 }
 
+export const createSharedMutateFn = (engine, mutationObj) =>
```

Comment:
Seems like this doesn't need exporting?

Reply:
True. At first I imported this in `use-query-client`, before I created a hook for setting the default functions. 
I guess that can be removed now. It's also the reason for splitting this out into a new function - so it could be used imperatively, outside of hooks. That's not strictly needed anymore, but I think it's clean, and it could be valuable in the future.